docs/operating-scylla/security/rbac-usecase.rst

```diff
@@ -87,9 +87,9 @@ If you proceed with the following example without authorization, you will see th
 
 .. code-block:: cql
 
-   CREATE KEYSPACE IF NOT EXISTS customer WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', 'replication_factor' : 1 };
+   CREATE KEYSPACE IF NOT EXISTS customer WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', 'replication_factor' : 3 };
```

Comment:
A quick comment: i'm not sure increasing RF doesn't impact the behavior of the test (?)

Reply:
I'm taking back what I said, this looks OK.